### PR TITLE
Chore/upgrade registrypackages build

### PR DESCRIPTION
**Pull request by KraMorK**
## Description

Reduce differences between cse and main branches. Standardize build, update module dependencies
## Why do we need it, and what problem does it solve?

Migrate cse changes to main branch
## Why do we need it in the patch release (if we do)?

not related to release
## What is the expected result?

Reduce differences between cse and main branches
## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

Migrate cse changes to main branch
```changes
section: registrypackages
type: chore
summary: Migrate cse changes to main branch.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've started the workflow for label 'e2e/run/azure'...

**Comment by KraMorK** [comment]
/e2e/abort azure;Standard;containerd;1.29 12236736021 12236736021-1-con-1-29 /sys/deckhouse-oss/install:pr11006 azureuser@20.86.138.250

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've started the workflow for label 'e2e/run/azure'...

File: modules/007-registrypackages/images/containerd/werf.inc.yaml [hunk 1]
```diff
@@ -2,10 +2,26 @@
 {{- $image_version := $containerd_version | replace "." "-" }}
 {{- $runc_version := "1.2.2" }}
 ---
+image: {{ .ModuleName }}/{{ .ImageName }}-src-artifact-{{ $image_version }}
+final: false
+fromArtifact: common/src-artifact
```
Comment (RomanenkoDenys) [inline modules/007-registrypackages/images/containerd/werf.inc.yaml:7]: fromImage

File: modules/007-registrypackages/images/crictl/werf.inc.yaml [hunk 2]
```diff
@@ -4,10 +4,25 @@
   {{- $crictlPatch := toString $value.crictlPatch }}
   {{- $full_version := printf "%s.%s" $version $crictlPatch }}
 ---
+image: {{ $.ModuleName }}/{{ $.ImageName }}-src-artifact-{{ $image_version }}
+final: false
+fromArtifact: common/src-artifact
```
Comment (RomanenkoDenys) [inline modules/007-registrypackages/images/crictl/werf.inc.yaml:9]: fromImage

File: modules/007-registrypackages/images/d8-curl/werf.inc.yaml [hunk 3]
```diff
@@ -1,10 +1,24 @@
-{{- $version := "8.2.1" }}
+{{- $version := "8.9.1" }}
 {{- $image_version := $version | replace "." "-" }}
 ---
+image: {{ $.ModuleName }}/{{ $.ImageName }}-src-artifact-{{ $image_version }}
+final: false
+fromArtifact: common/src-artifact
```
Comment (RomanenkoDenys) [inline modules/007-registrypackages/images/d8-curl/werf.inc.yaml:6]: fromImage

File: modules/007-registrypackages/images/d8/werf.inc.yaml [hunk 4]
```diff
@@ -1,8 +1,22 @@
 ---
+image: {{ $.ModuleName }}/{{ $.ImageName }}-src-artifact
+final: false
+fromArtifact: common/src-artifact
```
Comment (RomanenkoDenys) [inline modules/007-registrypackages/images/d8/werf.inc.yaml:4]: fromImage

File: modules/007-registrypackages/images/docker-registry/werf.inc.yaml [hunk 5]
```diff
@@ -3,10 +3,32 @@
 
 {{- $image_version := $docker_version | replace "." "-" }}
 ---
+image: {{ $.ModuleName }}/{{ $.ImageName }}-src-artifact-{{ $image_version }}
+final: false
+fromArtifact: common/src-artifact
```
Comment (RomanenkoDenys) [inline modules/007-registrypackages/images/docker-registry/werf.inc.yaml:8]: fromImage

File: modules/007-registrypackages/images/e2fsprogs/werf.inc.yaml [hunk 6]
```diff
@@ -1,10 +1,24 @@
 {{- $version := "1.47.0" }}
 {{- $image_version := $version | replace "." "-" }}
 ---
+image: {{ $.ModuleName }}/{{ $.ImageName }}-src-artifact-{{ $image_version }}
+final: false
+fromArtifact: common/src-artifact
```
Comment (RomanenkoDenys) [inline modules/007-registrypackages/images/e2fsprogs/werf.inc.yaml:6]: fromImage

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
some nits

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've started the workflow for label 'e2e/run/azure'...

**Comment by KraMorK** [comment]
/e2e/abort azure;Standard;containerd;1.29 12409758647 12409758647-1-con-1-29 /sys/deckhouse-oss/install:pr11006 azureuser@20.86.148.16

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've recognized your '/e2e/abort' command and started the workflow...

File: modules/007-registrypackages/images/socat/werf.inc.yaml [hunk 7]
```diff
@@ -17,24 +31,26 @@ docker:
...
-  - add: /{{ $.ModulePath }}modules/007-{{ $.ModuleName }}/images/{{ $.ImageName }}/scripts
-    to: /
-    stageDependencies:
-      setup:
-      - '**/*'
+image: {{ $.ModuleName }}/{{ $.ImageName }}-artifact-{{ $image_version }}
+final: false
+fromImage: common/alt-p11-artifact
+import:
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-src-artifact-{{ $image_version }}
```
Comment (sprait) [inline modules/007-registrypackages/images/socat/werf.inc.yaml:40]:
```suggestion
- image: {{ $.ModuleName }}/{{ $.ImageName }}-src-artifact-{{ $image_version }}
```

**Review by sprait — CHANGES_REQUESTED** [review]
one suggestion

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @KraMorK. I've started the workflow for label 'e2e/run/aws'...

File: modules/007-registrypackages/images/crictl/werf.inc.yaml [hunk 8]
```diff
@@ -20,13 +35,16 @@ docker:
...
 ---
-artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact-{{ $image_version }}
-{{- if semverCompare ">=1.30" $version }}
-from: {{ $.Images.BASE_GOLANG_22_ALPINE_DEV }}
-{{- else }}
-from: {{ $.Images.BASE_GOLANG_20_ALPINE_DEV }}
-{{- end }}
+image: {{ $.ModuleName }}/{{ $.ImageName }}-artifact-{{ $image_version }}
+final: false
+from: {{ $.Images.BASE_GOLANG_23_ALPINE_DEV }}
```
Comment (sprait) [inline modules/007-registrypackages/images/crictl/werf.inc.yaml:42]:
```suggestion
from: {{ $.Images.BASE_GOLANG_23_ALPINE }}
```

File: modules/007-registrypackages/images/docker-registry/werf.inc.yaml [hunk 9]
```diff
@@ -21,28 +43,35 @@ docker:
     version: all
     docker: {{ $docker_version }}
     cesanta_auth: {{ $docker_auth_version }}
+  USER: 64535
 ---
-artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact-{{ $image_version }}
+image: {{ $.ModuleName }}/{{ $.ImageName }}-artifact-{{ $image_version }}
+final: false
 from: {{ $.Images.BASE_GOLANG_22_ALPINE_DEV }}
```
Comment (sprait) [inline modules/007-registrypackages/images/docker-registry/werf.inc.yaml:50]:
```suggestion
from: {{ $.Images.BASE_GOLANG_23_ALPINE }}
```

File: werf.yaml [hunk 10]
```diff
@@ -212,7 +212,7 @@ import:
   to: /etc/ssl
   before: setup
 
-- artifact: registrypackages/d8-curl-artifact-8-2-1
+- artifact: registrypackages/d8-curl-artifact-8-9-1
```
Comment (sprait) [inline werf.yaml:215]:
```suggestion
- image: registrypackages/d8-curl-artifact-8-9-1
```

**Review by sprait — CHANGES_REQUESTED** [review]
some suggestions

**Review by sprait — APPROVED** [review]

**Review by name212 — APPROVED** [review]

**Pull request merged**